src/app/containers/FeaturesAnalysis/index.jsx

```diff
@@ -0,0 +1,99 @@
+import React, { useContext } from 'react';
+import { arrayOf, shape, bool, node } from 'prop-types';
+import SectionLabel from '@bbc/psammead-section-label';
+import styled from 'styled-components';
+import { StoryPromoLi, StoryPromoUl } from '@bbc/psammead-story-promo-list';
+import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import {
+  GEL_SPACING_DBL,
+  GEL_SPACING_TRPL,
+} from '@bbc/gel-foundations/spacings';
+
+import featuresAnalysis from '#pages/StoryPage/featuresAnalysis.json';
+import { RequestContext } from '#contexts/RequestContext';
+import { storyItem } from '#models/propTypes/storyItem';
+import { ServiceContext } from '#contexts/ServiceContext';
+import { GhostGrid, GridItemConstrainedLarge } from '#lib/styledGrid';
+import StoryPromo from '../StoryPromo';
+import DivHeadline from '../StoryPromo/DivHeadlines';
+
+const Wrapper = styled(GridItemConstrainedLarge)`
+  margin-bottom: ${GEL_SPACING_DBL};
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    margin-bottom: ${GEL_SPACING_TRPL};
+  }
+  /* z-index needs explicitly set as the psammead section label component uses negative z-indices */
+  z-index: 0;
+`;
+
+const StyledSectionLabel = styled(SectionLabel)`
+  margin-top: 0;
+
+  span {
+    background-color: transparent;
+  }
+`;
+
+const FeaturesAnalysis = ({ content, enableGridWrapper }) => {
```

Comment:
Do we need `enableGridWrapper` for this component? I think it was only needed for `CpsRelatedContent` because it was integrated differently in `MediaAsset` pages compared to `Story` pages.

Reply:
What does the grid wrapper do?

Reply:
Seen.

I've taken it out.